tools/test/resources/resource_test.py

```diff
@@ -0,0 +1,129 @@
+# mbed SDK
+# Copyright (c) 2019 ARM Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+from os.path import dirname, join
+from tools.resources import Resources, FileType
+from tools.notifier.mock import MockNotifier
+
+SRC_PATHS = {
+    '': join(dirname(__file__), 'source'),
+    'mbed-os': join(dirname(__file__), 'mbed-os.lib'),
```

Comment:
Can you document why `mbed-os.lib` is being mapped here? Doesn't it have to do with the online compiler?